admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -1081,6 +1081,20 @@ Available:
 
 Custom CSP policy, changing this will overwrite the standard policy
```

Comment:
@MorrisJobke but this sentence is not in the sample config.
So are you sure this file is the one that is being overwritten?